identity-service/src/routes/ethRelay.js

```diff
@@ -17,27 +17,34 @@ module.exports = function (app) {
           senderAddress: body.senderAddress,
           gasLimit: body.gasLimit || null
         }
-        receipt = await txRelay.sendEthTransaction(req, txProps, reqBodySHA)
-        return successResponse({ receipt })
+        receipt = await ethTxRelay.sendEthTransaction(req, txProps, reqBodySHA, function(txHash){
```

Comment:
@raymondjacobson i think this is what we talked about yesterday. sending the tx hash in a callback but awaiting the entire route until we get receipt confirmation from chain

Reply:
also @hareeshnagaraj i dont think having/not having a return inside the cb matters. i wrote this script to check. as long as the response goes to the client
```js
async function delay (ms) {
  return new Promise(resolve => setTimeout(resolve, ms))
}

async function test () {
  const v = await test2('hi', function(val) {
    console.log("got cb", val)
    return val
  })
  console.log("after", v)
}

async function test2 (val, cb) {
  cb('cb value')
  await delay(2000)
  return 'promise value'
}

test()
```

Reply:
Yeah I think this makes sense to me!

Reply:
yeah same here, sendResponse invoking should negate requirement for any return